sats-dna/src/main/kotlin/com/sats/dna/components/button/SatsDismissButton.kt

```diff
@@ -31,6 +32,7 @@ import com.sats.dna.theme.SatsTheme
 fun SatsDismissButton(
     dismissLabel: String,
     onDismissClicked: () -> Unit,
+    isLoading: Boolean,
```

Comment:
If we want this to be more in line with the general SatsButton function we would add isEnabled as an option here too. But idk if it makes sense to have a dismiss button that's disabled except for when it's in the process of loading 🤔

Reply:
Nah, there's no need for that, let's just get this merged first. :+1: